home-service/src/main/java/com/softserveinc/ita/homeproject/homeservice/mapper/ServiceMapper.java

```diff
@@ -0,0 +1,23 @@
+package com.softserveinc.ita.homeproject.homeservice.mapper;
+
+import com.softserveinc.ita.homeproject.homeservice.mapper.config.CustomServiceMappingConfig;
+import lombok.RequiredArgsConstructor;
+import org.modelmapper.ModelMapper;
+import org.springframework.stereotype.Component;
+
+import javax.annotation.PostConstruct;
+import java.util.Set;
+
+@Component
+@RequiredArgsConstructor
+public class ServiceMapper extends ModelMapper {
```

Comment:
Again. It would be better to use composition over inheritance here.

Reply:
Done.